### PR TITLE
chore: add get_by_doc_id API endpoint tests

### PR DESCRIPTION
**Pull request by stefanomunarini**

### CODE REVIEW

File: main/tests.py [hunk 1]
```diff
@@ -104,3 +108,15 @@ def test_fail_search_ref(self):
...
         self.assertEqual(results_count, 0)
+
+    def test_get_by_doc_id(self):
+        url = f'{reverse("api_get_by_docid")}?docid={self.ref_body.get("id")}'
+        response = self.client.get(url, **self.api_headers)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(json.loads(response.content)['data']['id'], self.ref_body['id'])
+
+    def test_not_found_by_doc_id(self):
+        url = f'{reverse("api_get_by_docid")}?docid=UNEXISTENT_DOC_ID'
```
Comment (ronaldtse) [inline main/tests.py:119]: "UNEXISTENT_DOC_ID" => "NONEXISTENT_DOC_ID"

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by strogonoff** [comment]
Let me take a look…

**Comment by ronaldtse** [comment]
@stefanomunarini did you mean to close this, or are you making a new PR?

**Comment by stefanomunarini** [comment]
@ronaldtse I've renamed the remote branch to a more generic **feat/api_tests** and the PR got closed automatically

**Pull request closed without merging**